src/tox_wheel/plugin.py

```diff
@@ -60,6 +64,13 @@ def patch(obj, attr, value):
         setattr(obj, attr, original)
 
 
+@hookimpl
+def tox_testenv_install_deps(venv, action):
+    if venv.envconfig.wheel_pep517 == "build":
+        venv.run_install_command(["build[virtualenv]==0.7.0"], action)
```

Comment:
This shouldn't be a pin as it can downgrade the package. Use a `>=`?

Reply:
You never know when someone needs a newer build version. Exact version pinning is a bad practice, you should avoid it in general for specifying library dependencies. More on that: https://caremad.io/posts/2013/07/setup-vs-requirement/

Reply:
Interesting article - I have made the change as requested.